src/components/shared/Icon/Icon.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { HTMLAttributes } from 'react';
 import { MessageDescriptor } from 'react-intl';
 
+import clsx from 'clsx';
```

Comment:
can we move this import up and separate the library imports from the webpack imports with a space

Reply:
@thomassdk done